apps/financial-remedy/finrem-cos/finrem-cos.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/finrem/cos:prod-beb9e4e-20220527092537 #{"$imagepolicy": "flux-system:finrem-cos"}
+      image: hmctspublic.azurecr.io/finrem/cos:pr-695-4fe3100-20220527084333 #{"$imagepolicy": "flux-system:demo-finrem-cos"}
```

Comment:
This should be {"$imagepolicy": "flux-system:demo-image-policy"}

Reply:
changed. thank you.